.gitignore

```diff
@@ -32,3 +32,6 @@ yarn-error.log
 
 # annoying mac file
 /.DS_Store
+
+# vscode `experimentalDecorators` syntax-highlighting
+tsconfig.json
```

Comment:
Oh cool, does this get rid of that annoying warning about using decorators?

Reply:
Yup 🙂 otherwise every function and property name in a component has those red underline squiggles.

Reply:
Oh wait, we should actually commit `tsconfig.json`, not ignore it....

Reply:
🤦‍♀️ oh woops. i potentially wanted to avoid conflicting with anyone else's config if they had it, but then they'd have to figure out how to do the same thing if they're using visual studio code. I'll add the config and remove this ignore statement